commit e7dd5e9a16fb48b83aa5127c21eaf5f3a17fc5c2
Author: Tarulia <mihawk.90+git@googlemail.com>
Date:   Sun Mar 30 06:18:47 2025 +0200

    deps/json11: Fix compile error on GCC 15+

diff --git a/deps/json11/json11.cpp b/deps/json11/json11.cpp
index 88024e922..45dbbcbe6 100644
--- a/deps/json11/json11.cpp
+++ b/deps/json11/json11.cpp
@@ -22,6 +22,7 @@
 #include "json11.hpp"
 #include <cassert>
 #include <cmath>
+#include <cstdint>
 #include <cstdlib>
 #include <cstdio>
 #include <limits>
