diff -uNrp a/UI/data/themes/Acri.qss b/UI/data/themes/Acri.qss
--- a/UI/data/themes/Acri.qss	2022-09-03 04:09:59.000000000 +0100
+++ b/UI/data/themes/Acri.qss	2022-09-13 13:58:16.263225325 +0100
@@ -300,8 +300,8 @@ QDockWidget {
     font-size: 10.5pt;
     font-weight: bold;
 
-    titlebar-close-icon: url('./Dark/Close.svg');
-    titlebar-normal-icon: url('./Dark/Popout.svg');
+    titlebar-close-icon: url('./Dark/close.svg');
+    titlebar-normal-icon: url('./Dark/popout.svg');
 }
 
 QDockWidget::title {
diff -uNrp a/UI/data/themes/Grey.qss b/UI/data/themes/Grey.qss
--- a/UI/data/themes/Grey.qss	2022-09-03 04:09:59.000000000 +0100
+++ b/UI/data/themes/Grey.qss	2022-09-13 13:58:16.265225322 +0100
@@ -300,8 +300,8 @@ QDockWidget {
     font-size: 10.5pt;
     font-weight: bold;
 
-    titlebar-close-icon: url('./Dark/Close.svg');
-    titlebar-normal-icon: url('./Dark/Popout.svg');
+    titlebar-close-icon: url('./Dark/close.svg');
+    titlebar-normal-icon: url('./Dark/popout.svg');
 }
 
 QDockWidget::title {
diff -uNrp a/UI/data/themes/Light.qss b/UI/data/themes/Light.qss
--- a/UI/data/themes/Light.qss	2022-09-03 04:09:59.000000000 +0100
+++ b/UI/data/themes/Light.qss	2022-09-13 13:58:16.267225319 +0100
@@ -300,8 +300,8 @@ QDockWidget {
     font-size: 10.5pt;
     font-weight: bold;
 
-    titlebar-close-icon: url('./Light/Close.svg');
-    titlebar-normal-icon: url('./Light/Popout.svg');
+    titlebar-close-icon: url('./Light/close.svg');
+    titlebar-normal-icon: url('./Light/popout.svg');
 }
 
 QDockWidget::title {
diff -uNrp a/UI/data/themes/Rachni.qss b/UI/data/themes/Rachni.qss
--- a/UI/data/themes/Rachni.qss	2022-09-03 04:09:59.000000000 +0100
+++ b/UI/data/themes/Rachni.qss	2022-09-13 13:58:16.269225316 +0100
@@ -302,8 +302,8 @@ QDockWidget {
     font-size: 10.5pt;
     font-weight: bold;
 
-    titlebar-close-icon: url('./Dark/Close.svg');
-    titlebar-normal-icon: url('./Dark/Popout.svg');
+    titlebar-close-icon: url('./Dark/close.svg');
+    titlebar-normal-icon: url('./Dark/popout.svg');
 }
 
 QDockWidget::title {
diff -uNrp a/UI/data/themes/Yami.qss b/UI/data/themes/Yami.qss
--- a/UI/data/themes/Yami.qss	2022-09-03 04:09:59.000000000 +0100
+++ b/UI/data/themes/Yami.qss	2022-09-13 13:58:16.271225313 +0100
@@ -304,8 +304,8 @@ QDockWidget {
     font-size: 10.5pt;
     font-weight: bold;
 
-    titlebar-close-icon: url('./Dark/Close.svg');
-    titlebar-normal-icon: url('./Dark/Popout.svg');
+    titlebar-close-icon: url('./Dark/close.svg');
+    titlebar-normal-icon: url('./Dark/popout.svg');
 }
 
 QDockWidget::title {
