--- plugins/obs-ffmpeg/obs-ffmpeg-mux.c
+++ plugins/obs-ffmpeg/obs-ffmpeg-mux.c
@@ -235,8 +235,11 @@
		aencoders[num_tracks] = aencoder;
		num_tracks++;
	}
-
+#ifdef FFMPEG_MUX_FIXED
+	dstr_init_copy(cmd, FFMPEG_MUX_FIXED);
+#else
	dstr_init_move_array(cmd, obs_module_file(FFMPEG_MUX));
+#endif
 	dstr_insert_ch(cmd, 0, '\"');
	dstr_cat(cmd, "\" \"");

--- CMakeLists.txt	2018-08-20 18:13:28.000000000 +0200
+++ CMakeLists.txt	2018-08-22 01:50:26.191160839 +0200
@@ -31,6 +31,7 @@
 include(ObsHelpers)
 include(ObsCpack)
 include(GNUInstallDirs)
+add_definitions(-DFFMPEG_MUX_FIXED=\"/usr/libexec/obs-plugins/obs-ffmpeg/ffmpeg-mux\")
 
 if(WIN32)
 	string(REPLACE "-" ";" UI_VERSION_SPLIT ${OBS_VERSION})
